internal/datastore/crdb/migrations/zz_migration.0009_add_expiration_support.go

```diff
@@ -0,0 +1,67 @@
+package migrations
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+
+	"github.com/Masterminds/semver"
+	"github.com/jackc/pgx/v5"
+)
+
+const (
+	addExpirationColumnToRelationTuple = `
+		ALTER TABLE relation_tuple
+		ADD COLUMN IF NOT EXISTS expires_at TIMESTAMPTZ DEFAULT NULL;
+
+		ALTER TABLE relation_tuple_with_integrity
+		ADD COLUMN IF NOT EXISTS expires_at TIMESTAMPTZ DEFAULT NULL;
+	`
+
+	// ttl_expiration_expression support was added in CRDB v22.2.
+	addExpirationPolicy = `
+		ALTER TABLE relation_tuple SET (ttl_expiration_expression = 'expires_at', ttl_job_cron = '@daily');
+
+		ALTER TABLE relation_tuple_with_integrity SET (ttl_expiration_expression = 'expires_at', ttl_job_cron = '@daily');
+	`
```

Comment:
any specific reason we don't want to garbage collect those more frequently? is daily the recommended value?

Reply:
Because it has to be outside of the GC window and we recommend GC windows no greater than 24 hours. We can't really support a "dynamic" option here